go.mod

```diff
@@ -174,7 +175,7 @@ require (
 	github.com/googleapis/enterprise-certificate-proxy v0.2.3 // indirect
 	github.com/googleapis/gax-go/v2 v2.11.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.16.0 // indirect
-	github.com/hashicorp/aws-sdk-go-base/v2 v2.0.0-beta.58 // indirect
+	github.com/hashicorp/aws-sdk-go-base/v2 v2.0.0-beta.62 // indirect
```

Comment:
Remote backends tend to have their own `go.mod`, is this something you could bump instead? https://github.com/hashicorp/terraform/blob/main/internal/backend/remote-state/s3/go.mod#L13